### PR TITLE
Remove _fireOnDomLoad

### PR DESCRIPTION
**Pull request by cea2aj**
We can remove the `_fireOnDomLoad` from the manual initializer because the setup function itself is called during `DOMContentLoaded`

J=none
TEST=manual

Smoke test runtime config and manual initialization

### CODE REVIEW

File: layouts/html.hbs [hunk 1]
```diff
@@ -6,6 +6,14 @@
       if (window.name == 'overlay') {
         window.isOverlay = true;
       }
+      function onDocumentReady(cb) {
+        if (document.readyState === "complete"
```
Comment (tmeyer2115) [inline layouts/html.hbs:10]: Is it possible we will ever invoke `onDocumentReady` when the document is already loaded? I'm wondering if we need this condition or if the `addEventListener` will suffice.

Comment (cea2aj) [reply layouts/html.hbs:10]: I'm using it inside the iframeResizer onMessage, so we do need to check the ready state. Also I think it makes it a little more robust overall

**Review by tmeyer2115 — APPROVED** [review]

File: layouts/html.hbs [hunk 2]
```diff
@@ -6,6 +6,14 @@
...
+          || document.readyState === "loaded") {
```
Comment (oshi97) [inline layouts/html.hbs:11]: nit can we do single quotes

Comment (cea2aj) [reply layouts/html.hbs]: sure. Updated!

**Review by oshi97 — APPROVED** [review]

**Review by tmeyer2115 — APPROVED** [review]

**Review by oshi97 — APPROVED** [review]

**Pull request merged**